tech_supp.qmd

```diff
@@ -485,17 +488,17 @@ base_map <-
     color = "black",
     weight = 3,
     fill = FALSE
-  ) 
+  )
+
+layers <- list(
+  "Presence/Absence" = fips_i_linear$FIPS_I_RoadsTracks_2,
+  "Distance" = fips_i_dist$FIPS_I_RoadsTracks_2,
+  "Proximity" = fips_i$FIPS_I_RoadsTracks_2
+)
```

Comment:
I simplified this interactive figure to display just one layer (original, distance, proximity).

I think this is enough to explain what's happening and prevents confusing looking maps with lots of overlapping layers.